app/org/sagebionetworks/bridge/redis/JedisTransaction.java

```diff
@@ -37,44 +34,32 @@ public JedisTransaction del(final String key) {
     }
 
     public List<Object> exec() {
-        try (Jedis jedis = this.jedis) {
+        try {
             return transaction.exec();
+        } finally {
+            close();
```

Comment:
No, the try-with-resources in the old code would call jedis.close(). We need to call this.close(), or we end up bypassing the "could not return resource" check.